go.mod

```diff
@@ -1,6 +1,6 @@
-module github.com/amaanq/coc.go
+module github.com/antonschnitzer/coc.go
```

Comment:
Why would you change this?

Reply:
Git noob here, I'm sorry. I didn't want to include this in the pull request. I forked your git and changed the module name to my git link so that I can use my forked git as dependency.